application/ui/src/features/inspect/footer/footer.component.tsx

```diff
@@ -26,13 +27,13 @@ const useCurrentJob = () => {
 };
 
 const useDefaultModel = () => {
-    const models = useTrainedModels();
+    const models = useCompletedModels();
     const { data: pipeline } = usePipeline();
     const { projectId } = useProjectIdentifier();
     const patchPipeline = usePatchPipeline(projectId);
 
     const hasSelectedModel = pipeline?.model?.id !== undefined;
-    const hasNonAvailableModels = models.length === 0;
+    const hasNonAvailableModels = isEmpty(models.filter(({ status }) => status === 'Completed'));
```

Comment:
I think since you are using `useCompletedModels` it's already should have complete status